backend/src/controllers/movies.test.js

```diff
@@ -1,10 +1,32 @@
-import { getMoviesByCategory, getMovieDetails } from 'modules/services/movies';
-import { fakeMovies, fakeMovie } from 'modules/utils/testUtils';
-import { getByCategory, getDetails } from './movies';
+import {
```

Comment:
Since we are importing several utils/functions from these files, I think we can do 
```suggestion
import * as MovieService from "modules/services/movies"
```
this applies to the other imports we have and for the usage it should be something like MovieService.getMoviesByCategory for example

Reply:
I applied it, thanks!